app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -6,6 +6,34 @@
   </p>
 <% elsif  @application_choice.offer_deferred? %>
   <p class="govuk-body govuk-body govuk-!-margin-top-2">
-    Your training will now start in <%= (@application_choice.course.start_date + 1.year).strftime('%B %Y') %>.
+  Your training will now start in <%= (@application_choice.course.start_date + 1.year).to_s(:month_and_year) %>.
+  </p>
+<% elsif @application_choice.pending_conditions? || @application_choice.recruited? || @application_choice.offer? %>
+  <p class="govuk-body govuk-body govuk-!-margin-top-2">
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          What to do if you’re unable to start training this year
```

Comment:
I think this is consistent with the images on the card but not the inline copy - it specifies _What to do if you’re unable to start training in {month year}_

Reply:
Thanks Steve, I think this was changed on the card today 👍  I'll make the changes now

Reply:
that was sneaky! looks good to me now